OVERVIEW.md

```
@@ -1,12 +1,26 @@
 ## Code Overview and Step Response Calculation
 
+## Table of Contents
+- [Code Overview and Step Response Calculation](#code-overview-and-step-response-calculation)
+  - [Configuration](#configuration)
+  - [Core Functionality](#core-functionality)
+  - [Argument Parsing (src/main.rs)](#argument-parsing-srcmainrs)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)